protfolio.html

```diff
@@ -0,0 +1,58 @@
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <title>
+
+    </title>
+</head>
+
+<body>
+    <hr class="big">
+    <header>
+        <div class="cent">
+            <a href="/Users/mims/Desktop/sei/homeworks/hw-w02d01-website/aboutMe.html"> about me</a>
```

Comment:
be careful with file locations as it will cause errors when you deploy your web applications online